### ISSUE TITLE
Display any app window in Symbiote

### ISSUE THREAD

**Issue by dhemery** [moredip/Frank#119]
I would like the ability to display windows other than the main window. Today a client and I had a desire to inspect an "input accessory" attached to the keyboard.  These are normal views, so it would be cool to be able to inspect and interact with the views.  But they exist in the keyboard window instead of the main window.

Displaying alternate windows would require not only changes in Symbiote, but also changes in Frank and the selector engines:
- Frank would need a way to identify and describe current windows.
- Frank would need a way to dump the contents of an identified alternate window.
- Selector engines would need a way to find views in alternate windows (a feature I've been meaning to add to Igor).

Given those features in Frank and selector engines, Symbiote could:
- Ask Frank for a list of windows
- Display an option box with the list of windows.
- Display the dump of whichever window the user selects.

I could add the required features to Frank and Igor, though I would probably have to leave Symbiote, UIQuery, and Shelley stuff to others.

**Comment by ondrejhanslik** [comment]
Shelley already has a way to inspect other windows because it searches views in all the windows. However, selector `view:'UIWindow'` won't return anything because the `view` selector doesn't include the starting view. Windows can be listed for example by `view parent view:'UIWindow'`

Lately I was doing some experiments with the keyboard and I am pretty sure the changes to dump all the windows would be easy. There is already the same code for Mac which dumps all the windows. For iOS only `keyWindow` hierarchy is dumped.

For the screenshot, frank has some interesting image capturing code, for example, if you type "http://localhost:37265/screenshot/allwindows", all the windows will be captured, including the keyboard.

In other words, the neccessary changes would be very small because almost everything is already there.
